diff --git a/src/dev/c0de/smartpgp/SmartPGPApplet.java b/src/dev/c0de/smartpgp/SmartPGPApplet.java
index c47870f..87162fb 100644
--- a/src/dev/c0de/smartpgp/SmartPGPApplet.java
+++ b/src/dev/c0de/smartpgp/SmartPGPApplet.java
@@ -1329,10 +1329,12 @@ public final class SmartPGPApplet extends Applet {
             return;
         }
 
+/*
         if(data.admin_pin.getTriesRemaining() <= 0) {
             data.isTerminated = true;
             return;
         }
+*/
 
         assertAdmin();
 
